test/controllers/NCTSMessageControllerSpec.scala

```diff
@@ -334,6 +342,7 @@ class NCTSMessageControllerSpec extends SpecBase with ScalaCheckPropertyChecks w
 
           val result = route(application, request).value
 
+          contentAsString(result) mustEqual "External Submission Failure Future(Success(SubmissionFailureExternal))"
```

Comment:
I think we should return a status code only here. It looks like SubmissionFailureExternal is being used for internal errors when we already have SubmissionFailureInternal for that. This needs to be reviewed.